### PR TITLE
CSS handling multiline code

### LINKED ISSUE #2649: CSS doesn't work with multiline text
this code
```
CSS(@"
    .my-style {
        color: #FF0000;
    }
");
```

does not work

### PR DESCRIPTION
**Pull request by colombod**
fixes #2649

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive/Kernel.Static.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ private static async Task<string> GetInputAsync(string prompt, bool isPassword,
     public static void CSS(string content) =>
         // From https://stackoverflow.com/questions/524696/how-to-create-a-style-tag-with-javascript
         Javascript($@"
-            var css = '{content}',
+            var css = `{content}`,
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive/Kernel.Static.cs:76]: I was considering removing this method altogether, along with the `Javascript` method. People should likely be emitting `HTML` elements for this or calling a JavaScript kernel if one is available.

Comment (colombod) [reply src/Microsoft.DotNet.Interactive/Kernel.Static.cs:76]: this is blocking some notebooks development as that is public api. I'd fix it and mark as deprecated?

Comment (colombod) [reply src/Microsoft.DotNet.Interactive/Kernel.Static.cs:76]: but I think is very convenient shortcut to be able to sue the `CSS` helper to add styles

Comment (jonsequitur) [reply src/Microsoft.DotNet.Interactive/Kernel.Static.cs:76]: Convenient isn't sufficient if it doesn't work consistently.

Comment (jonsequitur) [reply src/Microsoft.DotNet.Interactive/Kernel.Static.cs:76]: I'd suggest we leave it for now, with the fix, but should discuss whether this is the right design, especially given the Pythonesque top-level function name.

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**